nodejs/eks/package.json

```diff
@@ -12,8 +12,8 @@
     "repository": "https://github.com/pulumi/pulumi-eks",
     "dependencies": {
         "@pulumi/aws": "^0.16.0",
-        "@pulumi/kubernetes": "^0.18.0",
-        "@pulumi/pulumi": "^0.16.0",
+        "@pulumi/kubernetes": "^0.20.3",
+        "@pulumi/pulumi": "^0.16.14",
```

Comment:
Can you pin to `0.16.15`, which also has the `deleteBeforeReplace` property in `CustomResourceOptions`?

Reply:
:+1: